src/Steppenwolf/Startup.cs

```diff
@@ -57,7 +57,8 @@ public void ConfigureServices(IServiceCollection services)
                 .AddRoleManager<RoleManager<IdentityRole>>()
                 .AddEntityFrameworkStores<PostgresDbContext>();
 
-            services.AddServerSideBlazor(); // TODO change to client-side
+            services.AddServerSideBlazor() // TODO change to client-side
```

Comment:
Can't we move this into:
#if DEBUG
#else
#endif

Reply:
I'll see if it is possible to use environment setting here, don't particularly like the if DEBUG